### PR TITLE
Add sbt-ci-release

### PR DESCRIPTION
**Pull request by mlachkar**

### CODE REVIEW

File: scalafix/project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.2.1")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.6.0-M14-3")
 addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.2")
+addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.0")
```
Comment (bjaglin) [inline scalafix/project/plugins.sbt:4]: already added above

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: CI
+on:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v6
+      - run: sbt test
```
Comment (bjaglin) [inline .github/workflows/ci.yml:10]: won't that fail?
```suggestion
```

Comment (mlachkar) [reply .github/workflows/ci.yml:10]: Fails.. I fixed that

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: CI
+on:
+  push:
```
Comment (bjaglin) [inline .github/workflows/ci.yml:3]:
```suggestion
on: [pull_request]
```

File: scalafix/project/plugins.sbt (hunk 1 above)
Comment (mlachkar) [reply scalafix/project/plugins.sbt:4]: Sorry..Thanks

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name: CI
+on: [pull_request]
```
Comment (bjaglin) [inline .github/workflows/ci.yml:2]: mmm actually, sorry, but we also need master!
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```

**Pull request closed without merging**